engine/src/graphics/device.cpp

```diff
@@ -15,11 +15,11 @@
 namespace gen
 {
 
-	GraphicsDevice::GraphicsDevice(gen::Window & window)
+	GraphicsDevice::GraphicsDevice(std::string const & appName)
```

Comment:
It's fine for this PR but in the very next one you'll need a `GLFWwindow*` as well (to create a surface). 😄